src/api/get-installations.js

```diff
@@ -59,6 +64,8 @@ async function findWindowsBrowsers () {
     if (edgeAlias)
         installations['edge'] = edgeAlias;
 
+    await exec(`chcp ${origCodePage}`);
```

Comment:
In that case we should write same code after this line too. It's ok ?
[async function detectMicrosoftEdge () ](https://github.com/DevExpress/testcafe-browser-natives/blob/master/src/api/get-installations.js#L32)